src/main/kotlin/io/github/graphglue/model/FilterProperty.kt

```diff
@@ -1,5 +1,13 @@
 package io.github.graphglue.model
 
+import io.github.graphglue.connection.filter.TypeFilterDefinitionEntry
```

Comment:
why now? and where is it used?

Reply:
The import is necessary for the reference in the doc comment.
Without the import, I would have to use a fully qualified class name.